sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -290,8 +289,7 @@ export class ManagementClient {
           logErrorStackTrace(error);
           throw error;
         }
-      },
-      toSpanOptions(this._context.config)
+      }
```

Comment:
is it correct that these spans won't have host and entity name anymore? they are useful and we should not remove them

Reply:
Added back in tracing